components/modal/index.tsx

```diff
@@ -0,0 +1,104 @@
+import { Button } from '../';
+
+type ModalProps = {
+  visible: boolean;
+  onClose: () => void;
+};
+
+export default function Modal({ visible, onClose }: ModalProps): JSX.Element {
```

Comment:
I think this `Modal` component is specified to certain use case, please change the name corresponding to its functions, eg: `DownloadModal`

Reply:
You are right. will fix

Reply:
fixed. please check